data/net/minecraft/client/gui/screens/ChatScreen.mapping

```diff
@@ -10,8 +10,8 @@ CLASS net/minecraft/client/gui/screens/ChatScreen
 	METHOD chatPreviewTop (I)I
 		ARG 1 height
 	METHOD handleChatInput (Ljava/lang/String;Z)Z
-		ARG 1 message
-		ARG 2 addToChat
+		ARG 1 input
+		ARG 2 addToRecentChat
```

Comment:
This entire commit is bad. Do you have enigma/scribe open?

Reply:
I'm not sure what you mean by this.

Reply:
The commit reverts a lot of stuff you fixed

Reply:
Not sure how that happened but it should be fixed now